underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -126,6 +126,22 @@ public GCSUnderFileSystem(AlluxioURI uri) throws ServiceException {
     mBucketMode = GCSUtils.translateBucketAcl(acl, mAccountOwnerId);
   }
 
+  protected GCSUnderFileSystem(AlluxioURI uri,
```

Comment:
Could we make this constructor public and have the other constructor use `this(...)` to call this one?